### ISSUE TITLE
Placeholder is shown multiple times in empty editor

### ISSUE THREAD

**Issue by sauti** [ianstormtaylor/slate#2945]
#### Do you want to request a _feature_ or report a _bug_?

Bug

#### What's the current behavior?

Placeholder is shown multiple times if Enter is pressed in empty editor

![slate-placeholder](https://user-images.githubusercontent.com/13797718/61953014-0429b500-afbe-11e9-8298-67b938897f9a.gif)

#### What's the expected behavior?

Placeholder should be shown only once, when Enter is pressed in empty editor new placeholders should not appear

**Comment by steida** [comment]
I suppose the placeholder feature should be removed because it's very hard to do it properly.

**Comment by dannyvaughton** [comment]
I am struggling with this too. I am currently trying to implement my own placeholder component in the renderBlock function.

However, each component is unaware of each other, so I can't figure out how to update other placeholder components to hidden when the focus is not on them.

Does anyone know how to fix this?

**Comment by TechFollow** [comment]
You could try to use the `isFocused` attribute in your render method :
`    
renderInline(props, editor, next) {
    const { node, attributes, isFocused } = props
    ...
}`

**Comment by madhavhambar** [comment]
Hi, 
I would like to contribute. can you help me with PR

**Comment by ianstormtaylor** [comment]
I believe that this may be fixed by https://github.com/ianstormtaylor/slate/pull/3093, which has changed a *lot* of the logic in Slate and `slate-react` especially. I'm going to close this out, but as always, feel free to open a new issue if it persists for you. Thanks for understanding.
